packages/typespec-ts/test/modularIntegration/collectionFormat.spec.ts

```diff
@@ -17,11 +17,7 @@ describe("CollectionFormatClient Classical Client", () => {
   });
 
   it("should send multi format in query", async () => {
-    const result = await client.query.multi(["blue", "red", "green"], {
-      requestOptions: {
```

Comment:
Is this expected? I see there's no code change for this case?

Reply:
yes, once we switched to this we don't need to set `skipUrlEncoding: true` anymore.